draft-ietf-httpbis-origin-frame.md

```diff
@@ -72,6 +72,18 @@ as described in BCP 14 {{!RFC2119}} {{!RFC8174}} when, and only when, they appea
 capitals, as shown here.
 
 
+# The ORIGIN_SUPPORT HTTP/2 Setting {#setting}
+
+The ORIGIN_SUPPORT HTTP/2 setting {{!RFC7540}}, Section 6.5) indicate that a server
+desires the connection to be considered "opted in" to the behaviour described in this specification.
+
+The value of the ORIGIN_SUPPORT setting does not have any meaning. Note that support for this
+specification cannot be disabled once it is sent.
+
+ORIGIN_SUPPORT has no effect on servers, and MUST be ignored by them. Likewise, it MUST be ignored
+by clients who have configured a proxy.
+
```

Comment:
Please describe what semantics you want to associate with the *client* sending the setting.

Reply:
It has none, as above.

Reply:
Ahh, ignored by servers.  Yuck.  Can we just make the server generate an error instead?